packages/wdio-appium-service/README.md

```diff
@@ -86,9 +86,9 @@ export const config = {
 Map of arguments for the Appium server, passed directly to `appium`.
 
 See [the documentation](https://appium.github.io/appium.io/docs/en/writing-running-appium/server-args/index.html) for possible arguments.
-The arguments should be supplied in lower camel case, so `--pre-launch true` becomes `preLaunch: true` or passed as an array.
+The arguments are supplied in lower camel case. For instance, `debugLogSpacing: true` transforms into `--debug-log-spacing true`, or they can be supplied as outlined in the Appium documentation.
```

Comment:
Boolean flags should not have a value.
```suggestion
The arguments are supplied in lower camel case. For instance, `debugLogSpacing: true` transforms into `--debug-log-spacing`, or they can be supplied as outlined in the Appium documentation.
```

Can you link _Appium documentation_ to an Appium resource that could be of help here?

Reply:
Furthermore, the documentation link is connected in line 88; however, it points to the outdated Appium version. Would it be advisable to directly link to the documentation [here](https://github.com/appium/appium/blob/master/packages/appium/docs/en/cli/args.md) on GitHub?

Reply:
yes, that makes sense